client/README.md

```diff
@@ -121,7 +124,46 @@ dch -i  # increment release number
 dch -r  # create release
 ```
 
-Then you need to vendor the Rust dependencies:
+2. Versioning Scheme
+
+We follow the `MAJOR.MINOR.PATCH` semantic versioning convention for
+this package:
+
+* `MAJOR`: Incremented for incompatible API changes or significant functionality updates.
+* `MINOR`: Incremented for backward-compatible feature additions.
+* `PATCH`: Incremented for backward-compatible bug fixes.
+
+
+For pre-releases targeted at a PPA, append the `~ppaN` suffix to the
+version, where `N` represents the build number (e.g., `1.2.3~ppa1`). Once
+the package is approved for publication to the main repository, remove
+the `~ppaN` suffix to finalize the version.
+
+3. Create a Matching Git Tag
+
+Ensure the version is correctly tagged in Git. The tag should follow
+the format `vMAJOR.MINOR.PATCH` and must be annotated to the same
+commit as the changelog entry:
+
+```
+git tag -a v1.2.3 -m "Release version 1.2.3"
+git push origin v1.2.3
+```
+
+4. Update Cargo Versions
+
+If the new version includes changes to the MAJOR, MINOR, or PATCH
+version numbers, update the version fields in the following files:
```

Comment:
I am little confused. Is there any chance there would be a release that doesn't change at least one of these numbers?

Reply:
you can release an update to PPA with the `ppa` prefix, but if you do releases to Ubuntu main, you should increment the version number